config.cson

```diff
@@ -5240,8 +5240,10 @@ fileIcons:
 	Vue:
 		icon: "vue"
 		scope: "vue"
-		match: ".vue"
-		colour: "light-green"
+		match: [
+			["vue.config.js", "light-green"]
```

Comment:
This line needs to be

~~~cson
[/^vue\.config\.js$/i, "light-green", priority: 2]
~~~